### DISCUSSION TITLE
Implementing GCanvas

### DISCUSSION THREAD

**Discussion by matteobruni** [flyskywhy/react-native-gcanvas#18] [Q&A] [answered]
Hello @flyskywhy,

I'd like to use this library for a React Native [tsParticles](https://github.com/matteobruni/tsparticles) component but I don't know where to start.

I tried implementing this but without success, I tried another library too with the same result as you can see here: https://github.com/matteobruni/tsparticles/tree/react-native

I'd like to know if there's some documentation that could be useful to me to start this new project.

The existing `react-native` branch can be deleted, it's pretty old.

**Comment by flyskywhy** [comment, marked as answer]
Just release @flyskywhy/react-native-gcanvas@2.3.0 , include a commit 😛 

https://github.com/flyskywhy/react-native-gcanvas/commit/3d29bbe060df8899593d58b72aed5d31ae3f3335
README.md: Solve #18 with example of how to directly use [zdog](https://github.com/metafizzy/zdog) without modifying any code of itself

Maybe you can ref to the commit and success.
